app/models/video.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from app import db
+
+
+class Video(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer, default=0)
+    rentals = db.relationship('Rental', back_populates='video') 
+
+    def to_json(self):
+        return {
+        "id": self.id,
+        "title": self.title,
+        "release_date": self.release_date,
+        "total_inventory": self.total_inventory,
+        "available_inventory": self.available_inventory
+        }
```

Comment:
Similar to above you can compute `available_inventory`:

```suggestion
    available_inventory = db.Column(db.Integer, default=0)
    rentals = db.relationship('Rental', back_populates='video') 

    def to_json(self):
        return {
        "id": self.id,
        "title": self.title,
        "release_date": self.release_date,
        "total_inventory": self.total_inventory,
        "available_inventory": self.total_inventory - len([rental for rental in self.rentals if not rental.check_in_date])
        }
```